src/createElement.test.ts

```diff
@@ -7,4 +8,18 @@ describe("createElement", () => {
       "Unsupported lower-case element 'test' encountered, please make sure all your components start with an upper-case letter and are functions."
     );
   });
+
+  it("throws an error if a multiple children detected for promise element", () => {
+    expect(() =>
+      // @ts-expect-error - promise element test multiple promises as children
+      createElement(
+        MdPromise,
+        {},
+        new Promise((res) => res("Test")),
+        new Promise((res) => res("Test"))
```

Comment:
Please use `Promise.resolve("Test")` instead. Mostly it is shorter, and it took me a while to understand what `res` is :)

Reply:
Noted.